awx/lib/awx_display_callback/module.py

```diff
@@ -475,6 +475,15 @@ def v2_runner_retry(self, result):
         with self.capture_event_data('runner_retry', **event_data):
             super(BaseCallbackModule, self).v2_runner_retry(result)
 
+    def v2_runner_on_start(self, host, task):
+        event_data = dict(
+            host=host,
```

Comment:
Seems like this isn't serializiable, given Alan's comment.  Should we `str()` this or is there a `.name` property or something?

Reply:
I'll check up on it and see wtf it is.